clients/ui/frontend/src/app/pages/modelRegistry/screens/RegisterModel/RegisterModel.tsx

```diff
@@ -3,32 +3,30 @@ import {
   Breadcrumb,
   BreadcrumbItem,
   Form,
-  FormGroup,
-  FormHelperText,
-  HelperText,
-  HelperTextItem,
   PageSection,
   Stack,
   StackItem,
-  TextArea,
-  TextInput,
 } from '@patternfly/react-core';
 import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
 import { useParams, useNavigate } from 'react-router';
 import { Link } from 'react-router-dom';
-import FormFieldset from '~/app/pages/modelRegistry/screens/components/FormFieldset';
```

Comment:
Just missing the fieldsets that were added in https://github.com/kubeflow/model-registry/blob/979b1ed1f14702fe572e72ab52042cbf17e44648/clients/ui/frontend/src/app/pages/modelRegistry/screens/RegisterModel/RegisterModel.tsx

Reply:
Thanks, really good catch, I've just added it there.